message_sender/factory.py

```diff
@@ -47,6 +48,21 @@ def __init__(self, url, auth=None, from_addr=None, session=None,
         self.override_payload = override_payload
         self.strip_filepath = strip_filepath
 
+    def _get_filename(self, path):
+        """
+        This function gets the base filename from the path, if a language code
+        is present the filename will start from there.
+        """
+        match = re.search('[a-z]{2,3}_[A-Z]{2}', path)
```

Comment:
Hi! Code is looking great cake I just have a few questions, as part of my learning process:

Reply:
I see here that we are trying to search some pattern "path", but what are we trying to do with the '[a-z]{2,3}_[A-Z]{2}'

Reply:
@OMosimege the '[a-z]{2,3}_[A-Z]{2}' is a regular expression to match a language code in the file path. It's saying, find a substring matching the given pattern, e.g. 'en_US', 'en_GB' etc and return that substring as match.